study.md

```diff
@@ -95,11 +101,11 @@ would use to satisfy these requirements.
 In your own words, explain the role of application seed data.
 
 ```md
-<!-- your response here -->
+The purose of seed data is a quick way to place data in an database for testing purposes.
 ```
 
 Should you use seeds to create data to experiment with during development?
 
 ```md
-<!-- your response here -->
+YES because how are you going to test your applicaiton with out test data.
 ```
```

Comment:
I don't want to give a hard no here. It's possible, but there are better ways such as using dummy data then dumping it before you move to production.